setup.cfg

```diff
@@ -0,0 +1,40 @@
+[metadata]
+name = pytest-factoryboy
+description = Factory Boy support for pytest.
+long_description = file: README.rst, AUTHORS.rst, CHANGES.rst
+long_description_content_type = text/x-rst
+author = Oleg Pidsadnyi, Anatoly Bubenkov and others
+license = MIT license
+author_email = oleg.pidsadnyi@gmail.com
+url = https://github.com/pytest-dev/pytest-factoryboy
+version = attr: pytest_factoryboy.__version__
+classifiers =
+    Development Status :: 6 - Mature
+    Intended Audience :: Developers
+    License :: OSI Approved :: MIT License
+    Operating System :: POSIX
+    Operating System :: Microsoft :: Windows
+    Operating System :: MacOS :: MacOS X
+    Topic :: Software Development :: Testing
+    Topic :: Software Development :: Libraries
+    Topic :: Utilities
+    Programming Language :: Python :: 3
+    Programming Language :: Python :: 3 :: Only
```

Comment:
I wouldn't use this classifier, as it implies that the project would not be compatible with a hypothetical python 4

Reply:
this is to match pytest's config: https://github.com/pytest-dev/pytest/blob/master/setup.cfg#L19

Reply:
Why do we have to match pytest's config?

Reply:
As far as we know, it's not compatible with Python 4...

Reply:
I disagree with this, if we follow this logic then we should also add `python_requires<4`, and we don't do that.
Python 4 will probably not introduce incompatible changes like python 3 did.

Reply:
Please remove this classifier.